### PR TITLE
create a separate ID block (outside the plan runner)

### PR DESCRIPTION
**Pull request by YeZhao**

### CODE REVIEW

File: drake/examples/kuka_iiwa_arm/kuka_plan_runner.cc [hunk 1]
```diff
@@ -191,6 +167,8 @@ class RobotPlanRunner {
 
   lcm::LCM lcm_;
   const RigidBodyTree<double>& tree_;
+  //InverseDynamicsController& torqueCtrl_;
+
```
Comment (kuindersma) [inline drake/examples/kuka_iiwa_arm/kuka_plan_runner.cc]: Just delete this?

Comment (YeZhao) [reply drake/examples/kuka_iiwa_arm/kuka_plan_runner.cc]: Sure, I am creating a new ID program using LCM channels.

**Comment by kuindersma** [comment]
Actually, I'd like to merge in what you have here first (just clean up those two comments). You can push to this branch and reopen the pull request.

We might want to discuss what the LCM message structure should be for the plan runner --> controller link

**Comment by YeZhao** [comment]
Sounds good, I'll remind you when we meet. Right now, I am trying to modify the kuka_plan_runner and create a new kuka_inverse_dynamics_runner. The kuka_plan_runner subscribes the iiwa status channel and publishes an ID_input channel storing all the data used by kuka_inverse_dynamics_runner. kuka_inverse_dynamics_runner subscribes iiwa status channel and the ID_input channel sent from kuka_planner_runner. It basically replicates the block diagram you draw on the lab whiteboard. Do you think this looks like a reasonable plan at this moment?

**Comment by kuindersma** [comment]
I think we could generalize it a bit more than that. E.g., have a controller_reference message that contains utime, q_ref, v_ref, vdot_ref, u0. Different types of feedback controllers (ID being just one) could consume that message. I'll try to put together a more complete block diagram tomorrow

**Comment by YeZhao** [comment]
Got it. The current controller reference message includes most of the mentioned parameters except u0.

**Comment by YeZhao** [comment]
@kuindersma I just got one version of the new ID controller block (use LCM messages). Now the inverse dynamics controller (i.e., kuka_inverse_dynamics_runner) becomes a "true" separate block (listed in the kuka_sim.pmd). I test it in simulation and it looks working well. But when i run it in hardware, it does not track as expected. Will further debug tomorrow. 

The code is temporarily pushed to my repo. https://github.com/YeZhao/drake/commit/4f1f3c5bd94671550d45a2c9ee393e29cc652489

**Pull request closed without merging**